opentelemetry-resource-detectors/src/host.rs

```diff
@@ -56,18 +67,24 @@ impl Default for HostResourceDetector {
 #[cfg(test)]
 mod tests {
     use super::HostResourceDetector;
-    use opentelemetry::Key;
+    use opentelemetry::{Key, Value};
     use opentelemetry_sdk::resource::ResourceDetector;
     use std::time::Duration;
 
     #[test]
     fn test_host_resource_detector() {
         let resource = HostResourceDetector::default().detect(Duration::from_secs(0));
-        assert_eq!(resource.len(), 1);
+        assert_eq!(resource.len(), 2);
         assert!(resource
             .get(Key::from_static_str(
                 opentelemetry_semantic_conventions::resource::HOST_ID
             ))
-            .is_some())
+            .is_some());
+        assert_eq!(
+            resource.get(Key::from_static_str(
+                opentelemetry_semantic_conventions::resource::HOST_ARCH
+            )),
+            Some(Value::from("x86_64"))
```

Comment:
nit - do we need this test to run for windows too ?

Reply:
Definitely, I just added a platform agnostic test: https://github.com/open-telemetry/opentelemetry-rust-contrib/pull/59/commits/35b7860c16a03beb3b945e251028ebc22bdea4bc